### PR TITLE
nginx 1.21.3

### PR DESCRIPTION
**Pull request by bevanjkay**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by bevanjkay** [comment]
@carlocab Is there anything I can do to help here? Just trying to learn some different scenarios. Do we just need to give passenger a revision bump? and is there a way to test that locally?

**Comment by carlocab** [comment]
Yes, just give `passenger` a `revision` bump.

File: Formula/passenger.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class Passenger < Formula
   url "https://github.com/phusion/passenger/releases/download/release-6.0.10/passenger-6.0.10.tar.gz"
   sha256 "3eb2c52652c96a136cc69cf342e3a61a8dd0e92cdbe9310a7aae249ab1abe0d4"
   license "MIT"
-  revision 1
+  revision 2
```
Comment (carlocab) [inline Formula/passenger.rb:7]: Split this into a separate commit, please.

Comment (bevanjkay) [reply Formula/passenger.rb:7]: @carlocab Done 👍🏻

**Review by carlocab — APPROVED** [review]
